web/src/OverviewActionBar.stories.tsx

```diff
@@ -43,7 +51,11 @@ export default {
   },
 }
 
-let defaultFilter = { source: FilterSource.all, level: FilterLevel.all }
+let defaultFilter: FilterSet = {
+  source: FilterSource.all,
+  level: FilterLevel.all,
+  term: EMPTY_FILTER_TERM,
+}
```

Comment:
@nicks, lmk if you'd like to see any other storybook changes made! The filter term input field is functional, though we don't have a view where the filters change what logs display.